### PR TITLE
D3ASIM-3209: Endpoint for configuration's registry

### PR DESCRIPTION
**Pull request by fievelk**
Create a method for `Aggregator` to reach the d3a-web endpoint for the aggregator's connected assets.

### CODE REVIEW

File: d3a_api_client/utils.py [hunk 1]
```diff
@@ -136,18 +136,23 @@ def request_response_returns_http_2xx(endpoint, resp):
...
 def blocking_get_request(endpoint, data, jwt_token):
     data["transaction_id"] = str(uuid.uuid4())
     resp = requests.get(
         endpoint,
         data=json.dumps(data),
         headers={"Content-Type": "application/json",
                  "Authorization": f"JWT {jwt_token}"})
-    return json.loads(resp.json()) if request_response_returns_http_2xx(endpoint, resp) else None
+
+    return resp.json() if request_response_returns_http_2xx(endpoint, resp) else None
```
Comment (hannesdiedrich) [inline d3a_api_client/utils.py:155]: Thanks! Didn't know that `requests` has a build-in json decoder.

**Review by hannesdiedrich — APPROVED** [review]
LGTM

File: README.md [hunk 2]
```diff
@@ -209,8 +217,28 @@ The device or market can unselect the Aggregator:
 device.unselect_aggregator(aggregator.aggregator_uuid)
 ```
 
+#### Available Aggregator's methods
+
+`Aggregator` instances provide methods that can simplify specific operations. Below we list some of the most commonly used:
+
+- Return all the aggregators connected to the aggregator's simulation: 
+    ```python
+    list_aggregators()
```
Comment (Hassan754) [inline README.md:226]: As a side comment, the list-aggregatorrs should not reside in an aggregator instance, it should be a standalone function

**Review by Hassan754 — APPROVED** [review]
LGTM

File: README.md (hunk 2 above)
Comment (fievelk) [reply README.md:226]: I agree. I suggest we create a task for this.

**Pull request merged**